scrapy_zyte_smartproxy/middleware.py

```diff
@@ -220,6 +220,10 @@ def _translate_headers(self, request, targets_zyte_api):
                 request,
             )
 
+    def _inc_stat(self, stat, value=1, /, *, targets_zyte_api):
```

Comment:
This breaks on 3.7 apparently

Reply:
Yes. I did it knowingly, thinking this repo had dropped 3.7- support already :facepalm:

Reply:
Hey! It seems the main change in this PR is actually dropping old Python support :) How hard would it be to keep it, and do separately?

Reply:
I think we wanted to be pretty conservative with this package.